client/cody-shared/src/chat/recipes/context-search.ts

```diff
@@ -71,10 +73,8 @@ export class ContextSearch implements Recipe {
             }
             let uri = new URL(`/search?q=context:global+file:${file.fileName}`, endpointUri).href
 
-            // TODO: Open file in editor (the uri is currently being stripped by the chat component)
-            // This current does not work hence the wsRootPath === uri
-            if (wsRootPath === uri) {
-                const vsceUri = vscode.Uri.parse('vscode://file:' + wsRootPath + '/' + file.fileName).toString()
+            if (wsRootPath != null) {
+                const vsceUri = vscode.Uri.parse('vscode://file:' + path.join(wsRootPath, file.fileName)).toString()
```

Comment:
```suggestion
                const fileUri = vscode.Uri.joinPath(vscode.Uri.file(wsRootPath), file.fileName)
```
Nite: we should use the vscode API for  it to work on all VS Code platforms 😄